app/views/users/index.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+<main>
+<% @users.each do |user|%>
+  <div class="container">
```

Comment:
- [ ] It would be better if we can navigate to the user show pages when we click on the user in the index page
![Screenshot from 2022-12-01 00-26-20](https://user-images.githubusercontent.com/46388113/204937049-dbaeba13-fae4-4cd3-bd18-88b6b44cebe8.png)
![Screenshot from 2022-12-01 00-29-05](https://user-images.githubusercontent.com/46388113/204937054-68cafda4-ab4c-44a2-a538-8c820bdb3be1.png)

Reply:
Resolved as requested